CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - `pyrgg.py` renamed to `graph_gen.py`
 - Other functions moved to `functions.py`
 - Test system modified
+- Refactor `params.py`
```

Comment:
Thanks for your effort, everything is good 💯 
Only need a minor edit, please write this line in our `CHANGELOG.md` format :
```
`params.py` refactored 
```
After this edit I will merge this PR

Best Regards